crypto/libsodium-fork/src/libsodium/crypto_core/ed25519/ref10/ed25519_ref10.c

```diff
@@ -999,6 +1195,18 @@ ge25519_is_canonical(const unsigned char *s)
     return 1 - (c & d & 1);
 }
 
+
+int ge25519_is_neutral_vartime(const ge25519_p3 *p) {
```

Comment:
the performance of this method could be improved; the call to `fe25519_tobytes` along with the temporary static memory writings could be completely avoided.

Reply:
this function is being called only once in each batch. so we won't gain a lot from trying to optimize it.